exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,7 @@
-var numberOfStudents = 15;
-var numberOfMentors = 8;
+let numberOfStudents = 15;
+let numberOfMentors = 8;
+let total=numberOfMentors+numberOfStudents
+let percentageOfStudent=numberOfStudents/total*100;
+console.log("Percentage students: "+Math.round(percentageOfStudent)+"%");
+let PercentageMentors=100-percentageOfStudent
```

Comment:
You could have also calculated the percentage of mentors, as.
`let PercentageMentors=numberOfMentors/total*100;`
and then use round. Surprisingly, this may not always be the same as the way you calculated it, because if the fraction ends in 0.5, it will always go up to the next integer when using Math.round. This is why in general, it is important to be aware and careful when rounding up numbers.

Reply:
I got it thanks.